libats-slick/src/main/scala/com/advancedtelematic/libats/slick/db/BootMigrations.scala

```diff
@@ -45,8 +45,11 @@ protected [db] object RunMigrations {
     val user = config.getString("database.properties.user")
     val password = config.getString("database.properties.password")
     val schemaO = Either.catchOnly[ConfigException.Missing](config.getString("database.catalog")).toOption
+    val schemaTable = Either.catchOnly[ConfigException.Missing](
+      config.getString("database.schema-table")
```

Comment:
we should add this to `reference.conf`

Reply:
As in, set the fallback at `ats.database.schema-version` instead of here?

Reply:
ah yes, that would be better, just so that all vars are documented in `reference.conf`

Reply:
Updated.